Foods.yaml

```
@@ -6,4 +6,5 @@
 
 FavouriteFoods:
   # Add the food to this list on this line, if possible.
-  - ["pizza", "kale_smoothies", "cheese", "pie"]
+  - ["Pizza", "Nachos", "kale_smoothies", "cheese", "pie"]
```

We should stick to `snake_case` for `yaml` files in the nubots codebase (as was pointed out to me yesterday lol)
```suggestion
  - ["pizza", "nachos", "kale_smoothies", "cheese", "pie"]
```